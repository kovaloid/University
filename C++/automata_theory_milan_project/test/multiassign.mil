BEGIN
	a:=x:=3;
    i := b := c := d := a := f := g := a + 2;
	WRITE(i);
	WRITE(a);
	WRITE(f);
	WRITE(c)
END
