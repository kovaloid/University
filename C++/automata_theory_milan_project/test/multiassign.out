0:	PUSH	3
1:	STORE	1
2:	LOAD	1
3:	STORE	0
4:	LOAD	0
5:	PUSH	2
6:	ADD
7:	STORE	7
8:	LOAD	7
9:	STORE	6
10:	LOAD	7
11:	STORE	0
12:	LOAD	7
13:	STORE	5
14:	LOAD	7
15:	STORE	4
16:	LOAD	7
17:	STORE	3
18:	LOAD	7
19:	STORE	2
20:	LOAD	2
21:	PRINT
22:	LOAD	0
23:	PRINT
24:	LOAD	6
25:	PRINT
26:	LOAD	4
27:	PRINT
28:	STOP
